app/src/androidTest/java/org/sagebionetworks/research/mpower/MainFragmentTest.java

```diff
@@ -20,7 +20,6 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.sagebionetworks.bridge.android.SingleFragmentActivity;
```

Comment:
This was breaking unit tests as the class didn't exist anymore.  So I found the class in a version of BridgeAndroidSdk and brought it in.  Is that OK?